learningresources/views.py

```diff
@@ -33,6 +37,13 @@ def welcome(request):
 
 
 @login_required
+@permission_required_or_403(
+    '{}.add_{}'.format(
```

Comment:
Is using internal app or model names good practice here? It may be better to just write in the name here since I think it only matters if it's consistent with other permission references